content/smoketest.md

```diff
@@ -232,26 +223,35 @@ character. The square brackets contain the image's alt text. Try to always use
 alt text so that people using screen readers can get some benefit from the
 image.
 
-![pencil icon](/images/pencil.png)
+![O3DE icon](/img/logos/O3DE-Circle-LogoMark-REV-MONO.svg)
+
+
+The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images and `.svg` diagrams don't scale overly large in wide browser windows.
+
+`image-width` takes three double-quoted parameters in order:
 
-To specify extended attributes, such as width, title, caption, etc, use the
-<a href="https://gohugo.io/content-management/shortcodes/#figure">figure shortcode</a>,
-which is preferred to using a HTML `<img>` tag. Also, if you need the image to
-also be a hyperlink, use the `link` attribute, rather than wrapping the whole
-figure in Markdown link syntax as shown below.
+1. image link
+1. width
+1. alt text
 
-{{< figure src="/images/pencil.png" title="Pencil icon" caption="Image used to illustrate the figure shortcode" width="200px" >}}
+`image-width` example:
 
-Even if you choose not to use the figure shortcode, an image can also be a link. This
-time the pencil icon links to the Kubernetes website. Outer square brackets enclose
+```markdown
+{{</* image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." */>}}
+```
+
+`image-width` example output:
+
+{{< image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." >}}
```

Comment:
This example reminds me that I think we should have named parameters in this shortcode.